### ISSUE TITLE
sd --inplace is changing the file's mode

### ISSUE THREAD

**Issue by vitiral** [chmln/sd#36]
```
mkdir play
cd play
touch foo.txt
touch bar.sh
chmod a+x bar.sh
git init
git add *
git commit -a -m "example"
sd -i a b foo.txt bar.sh
git diff
```

This is the output
```
diff --git a/bar.sh b/bar.sh
old mode 100755
new mode 100644
```

**Comment by chmln** [comment]
@vitiral this is a known issue. The good news is that it's already fixed on master :) if you want it right now, you can compile from master. Otherwise, just wait a day or two

**Comment by chmln** [comment]
Released in v0.6

**Comment by vitiral** [comment]
Thanks!
